apps/studio/components/ui/ErrorBoundaryState.tsx

```diff
@@ -19,6 +21,34 @@ export const ErrorBoundaryState = ({ error, resetErrorBoundary }: FallbackProps)
         <p className="text-sm text-foreground-light">Error: {error.message}</p>
       </div>
 
+      {isRemoveChildError && (
+        <Alert_Shadcn_ className="w-[650px]">
+          <WarningIcon />
+          <AlertTitle_Shadcn_>
+            This error might be caused by Google translate or third-party browser extensions
+          </AlertTitle_Shadcn_>
+          <AlertDescription_Shadcn_>
+            You may try to avoid using Google translate or disable certain browser extensions to
+            avoid running into the{' '}
+            <code className="text-xs">
+              {isRemoveChildError ? `'removeChild' on 'Node'` : `'insertBefore' on 'Node'`}
```

Comment:
Is this check needed? We're already inside `isRemoveChildError` right?
```suggestion
              'removeChild' on 'Node'
```

Reply:
oh yes sorry! forgot about that